### DISCUSSION TITLE
24.6.4

### DISCUSSION THREAD

**Discussion by davidjgraph** [jgraph/drawio-desktop#1800] [General]
## Releases Notes for 24.6.4
[Windows Installer](https://github.com/jgraph/drawio-desktop/releases/download/v24.6.4/draw.io-24.6.4-windows-installer.exe)
[Windows No Installer](https://github.com/jgraph/drawio-desktop/releases/download/v24.6.4/draw.io-24.6.4-windows-no-installer.exe)
[macOS - Universal](https://github.com/jgraph/drawio-desktop/releases/download/v24.6.4/draw.io-universal-24.6.4.dmg)
Linux - [deb](https://github.com/jgraph/drawio-desktop/releases/download/v24.6.4/drawio-amd64-24.6.4.deb), [AppImage](https://github.com/jgraph/drawio-desktop/releases/download/v24.6.4/drawio-x86_64-24.6.4.AppImage) or [rpm](https://github.com/jgraph/drawio-desktop/releases/download/v24.6.4/drawio-x86_64-24.6.4.rpm)

Windows intel x32 releases are marked -ia32-

**ChangeLog:**

- #1783 , #1784 , #1792
- Uses electron 30.0.6.
- Updates to [draw.io core 24.6.4](https://github.com/jgraph/drawio/blob/v24.6.4/ChangeLog). All changes from 24.6.2 to 24.6.4 are added in this build.

<hr /><em>This discussion was created from the release <a href='https://github.com/jgraph/drawio-desktop/releases/tag/v24.6.4'>24.6.4</a>.</em>

**Comment by akinuri** [comment]
Great! This solves the #1783.

**Comment by lexxy2422** [comment]
This is very lovely

**Comment by Siddhnak** [comment]
Facing issue..
Freezing every now and then when using stylus to take notes.
Version 24.6.4

Here's the proof
https://youtu.be/cB5qdLLIiRA
jump to 1:00.

**Comment by Chloe-oss** [comment]
How to download on iPad?

**Reply by alderg** [reply]
See https://www.drawio.com/doc/faq/mobile-diagram-app
